Source/GSAutoLayoutEngine.m

```diff
@@ -1,60 +1,460 @@
 /* Copyright (C) 2023 Free Software Foundation, Inc.
-   
+
    By: Benjamin Johnson
    Date: 28-2-2023
    This file is part of the GNUstep Library.
-   
+
    This library is free software; you can redistribute it and/or
    modify it under the terms of the GNU Lesser General Public
    License as published by the Free Software Foundation; either
    version 2.1 of the License, or (at your option) any later version.
-   
+
    This library is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
    Lesser General Public License for more details.
-   
+
    You should have received a copy of the GNU Lesser General Public
    License along with this library; if not, write to the Free
    Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
    Boston, MA 02110 USA.
 */
 
 #import "GSAutoLayoutEngine.h"
-#include "AppKit/NSLayoutConstraint.h"
+#import "AppKit/NSLayoutConstraint.h"
+#import "GSCSConstraint.h"
+#import "GSFastEnumeration.h"
+#import "NSViewPrivate.h"
+
+@interface
+GSAutoLayoutEngine (PrivateMethods)
```

Comment:
There is no need for the line break here.

Reply:
Fix